libs/SalesforceSDK/res/values/sf__styles.xml

```diff
@@ -3,53 +3,122 @@
 <resources xmlns:android="http://schemas.android.com/apk/res/android">
 
     <style name="SalesforceSDK">
+        <item name="android:actionBarStyle">@style/SalesforceSDKActionBar</item>
+        <item name="android:colorPrimary">@color/sf__primary_color</item>
+        <item name="android:colorPrimaryDark">@color/sf__primary_color</item>
+        <item name="sfPasscodeColorSecondary">@color/sf__secondary_color</item>-->
+        <item name="sfPasscodeLayoutBackground">@color/sf__layout_background</item>
+        <item name="android:colorBackground">@color/sf__background</item>
+        <item name="android:textColor">@color/sf__text_color</item>
+        <item name="android:textColorPrimary">@color/sf__text_color</item>
+        <item name="android:textColorHint">@color/sf__hint_color</item>
+        <item name="sfPasscodeBoarderColor">@color/sf__border_color</item>
+        <item name="android:itemBackground">@color/sf__background</item>
+        <item name="android:windowDrawsSystemBarBackgrounds">true</item>
+        <item name="android:navigationBarColor">@color/sf__background</item>
     </style>
 
-    <style name="SalesforceSDK.ActionBarTheme"
-        parent="@android:style/Theme.Holo.Light.DarkActionBar">
-        <item name="android:actionBarStyle">@style/SalesforceSDKActionBar</item>
+    <style name="SalesforceSDK.AccessibleNav">
+        <item name="android:navigationBarColor">@color/sf__accessibility_nav_color</item>
     </style>
 
-    <style name="SalesforceSDKActionBar"
-        parent="@android:style/Widget.Holo.Light.ActionBar.Solid.Inverse">
-        <item name="android:background">@drawable/sf__header_bg</item>
+    <style name="SalesforceSDK_Dark">
+        <item name="android:actionBarStyle">@style/SalesforceSDKActionBar_Dark</item>
+        <item name="android:colorPrimary">@color/sf__primary_color</item>
+        <item name="android:colorPrimaryDark">@color/sf__primary_color</item>
+        <item name="sfPasscodeColorSecondary">@color/sf__secondary_color_dark</item>
```

Comment:
I'm a little conflicted about this. All the items in here that are `SF` specific appear to be for the passcode screen. If so, they should probably be in a sub-theme, `SalesforceSDK.Passcode` or something.

Reply:
Good catch.  I just went through and renamed/deleted so that there is no passcode specific elements in the general theme.